RELEASE_NOTES.md

```diff
@@ -151,7 +153,7 @@ accompanying blog post.
 ## Bug Fixes
 
 - Properly use the display color profile in the slideshow. This was
-  missing and so images where just ignoring the color profile and were
+  missing, and so images were just ignoring the color profile and were
```

Comment:
Perhaps:
...missing, **so the the images' color profiles were ignored. Thus,** they were displayed (very) differently...

Reply:
Again, you can try to fix the precise words in the sentences if you like, but I'll probably rewrite a lot of them later anyway. For now let's just be sure that I'll understand what's meant, for when I copy-edit.